lib/ndr_import/mapper.rb

```diff
@@ -222,6 +230,20 @@ def validate_line_mappings(line_mappings)
     true
   end
 
+  # Apply ActiveRecord-like validations specified in field mappings, e.g.
+  # - column: column_one
+  #   mappings:
+  #   - field: field_one
+  #     validates:
+  #       presence: true
+  def apply_validations_on(field, value, validations)
+    presence_validation_on(field, value) if validations[Strings::PRESENCE]
+  end
+
+  def presence_validation_on(field, value)
+    raise "#{field} #{I18n.t('errors.messages.blank')}" if value.blank?
```

Comment:
I believe these translations are provided by `activemodel` / `activerecord`, which are currently implicit dependencies via `ndr_support`. Maybe this is ok?